protos/follow_me/follow_me.proto

```diff
@@ -59,13 +38,13 @@ message Config {
         FOLLOW_ALTITUDE_MODE_TARGET_GPS = 2; // Target GPS altitude taken into account to do 3D tracking
     }
 
-    float min_height_m = 1 [(mavsdk.options.default_value)="8.0"]; // Follow height in meters (recommended minimum 8 meters)
-    float follow_distance_m = 2 [(mavsdk.options.default_value)="8.0"]; // Follow distance to target in meters (recommended minimum 4 meter)
-    FollowDirection follow_direction = 3; // [!DEPRECIATED!] Direction to follow target at. Exists as a legacy to old PX4 Follow-Me, but now only used for setting the follow_angle_deg. This value never gets sent to the PX4 side!
+    float min_height_m = 1 [(mavsdk.options.default_value)="8.0"]; // [m] Follow height in meters (recommended minimum 8 meters)
+    float follow_distance_m = 2 [(mavsdk.options.default_value)="8.0"]; // [m] Follow distance to target in meters (recommended minimum 4 meter)
+    // FollowDirection follow_direction = 3; // [!DEPRECIATED!] Direction to follow target at. Exists as a legacy to old PX4 Follow-Me, but now only used for setting the follow_angle_deg. This value never gets sent to the PX4 side!
```

Comment:
Yeah makes sense to keep it commented out, so that we know that we should not reuse id 3!